### PR TITLE
add kubernetes deployment management

### PR DESCRIPTION
**Pull request by joker-bai**
# PR Checklist

Please post the link to the PR-related issue here (请将 PR 相关的 issues 的链接贴于此处):

Please make sure you've completed the relevant tasks for this PR from the following list (请确保完成下方检查项，提交后打勾):

- [x] The structure of the lab/challenge is correct.(实验/挑战的结构正确)
- [x] The format of the lab/challenge is correct.(实验/挑战的格式正确)
- [x] The code and verification scripts have been tested in the environment.(代码和验证脚本在环境中测试通过)
- [x] I promise that I have not plagiarised and have not copied the solution from other sources, AIGC is not included.(我保证没有抄袭，也没有从其他来源复制解决方案，AIGC 不包括在内)

See the [Contribution Guidelines](https://labex-docs.vercel.app/) for more details.(请参阅 [贡献指南](https://labex-docs.vercel.app/) 了解更多信息。)

### CODE REVIEW

File: kubernetes/challenge-kubernetes-deployment-management/step1.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+   - The password for root is `root`.
+   - Create a database named `hello-world`.
+   - Create a service called `mysql`.
+   - MySQL version is 5.7.
+
+     ![challenge-kubernetes-deployment-management-1-1](assets/challenge-kubernetes-deployment-management-1-1.png)
+
+2. Deploy the Hello World application with specific requirements.
+
+   - Use the `baidjay/hello-world:v2.0` image.
```
Comment (huhuhang) [inline kubernetes/challenge-kubernetes-deployment-management/step1.md:26]: 这个镜像需要放到我们的账号下面，避免未来失效。

Comment (huhuhang) [reply kubernetes/challenge-kubernetes-deployment-management/step1.md:26]: 账号密码已发。

**Review by huhuhang — CHANGES_REQUESTED** [review]

**Comment by joker-bai** [comment]
已提交

**Review by huhuhang — APPROVED** [review]

**Comment by huhuhang** [comment]
Hi, @joker-bai 

该 PR 未检测到正确关联 Issue，请你在 PR 描述中按要求添加，如有问题请及时联系 LabEx 的同事。

**Pull request merged**